packages/runtime-common/helpers/code-equality-assertion.ts

```diff
@@ -47,22 +47,18 @@ function standardize(code: string) {
 function preprocessTemplateTags(code: string): string {
   let output = [];
   let offset = 0;
-  let matches = parseTemplates(code, 'no-filename', 'template');
+  let matches = new ContentTagGlobal.Preprocessor().parse(code);
```

Comment:
@ef4 Looks like we need this not only in tests, but also in the app. We attempt to run this in module-syntax.ts via the ModuleContentsResource when loading a module in code mode.